src/pages/patterns/login-pattern/index.mdx

```diff
@@ -23,31 +23,27 @@ The login page allows a user to gain access to an application by entering their
+Login is often the first interaction a user has with your product. This entry point experience is an important moment in establishing your product's brand and experience and sets the tone for their overall experience with the product.
 
-| Term                 | Definition                                                                                                  |
-| -------------------- | ----------------------------------------------------------------------------------------------------------- |
-| Federated users      | IBM employees and other enterprise users who use single sign-on (SSO) to quickly log in.                    |
-| Non-federated users  | Other users who have a unique username and password to access the product.                                  |
-| Single sign-on (SSO) | Directs users to their company’s enterprise sign on and authenticates them using their company credentials. |
-
-### Anatomy
+### Anatomy of a basic login screen
 
 <Row>
 <Column colLg={12}>
 
-![Anatomy of a login page](/images/Login_anatomy.png)
+![Anatomy of a login page](./images/1_Login_anatomy_1120.png)
 
 </Column>
 </Row>
 
-1. **Title:** Log in to IBM [Product name].
-1. **User ID:** User IDs are usually in the form of an email address. Depending on the product, this area may also include a filter to specify the ID type.
-1. **Forgot user ID?:** Help link specifically for user ID questions and issues.
-1. **Remember me option**: Saves the user ID and presents the completed input field the next time the user logs in.
-1. **Continue button**: The primary call to action. When clicked, the email address is validated and routes the user to either the federated or non-federated flow.
-1. **Create account option:** Link to URX form to create an account.
-1. **Need help?:** Help link specifically for user ID questions and issues.
+1. **Title:** Located at the top of the log in flow. For consistency, the title should include the words “Log in,” rather than “Sign in” or another variant. The title can also include the product name if that makes sense for your situation. If necessary the title can wrap to the next line.
+2. **Create account (optional):** Link to URX form to create an account; location of this element can vary with layout.
+3. **Required fields:** The user ID and password fields are both required although, in IBM’s preferred log in flow, the password field is progressively disclosed, because it has a dependency on the user ID. User IDs are usually in the form of an email address. Depending on the product, this area may also include a filter to specify the ID type.
+4. **Forgot username/password link:** Takes users to a page where they can recover their username and/or password.
+5. **Remember ID (optional):** Saves the user ID and presents the completed input field the next time the user logs in; it is located under the required field. Clarify what is being remembered (i.e. “user ID") to avoid confusion.
```

Comment:
```suggestion
5. **Remember ID (optional):** Saves the user ID and presents the completed input field the next time the user logs in; it is located under the required field. Clarify what is being remembered (that is, “user ID") to avoid confusion.
```